App.js

```diff
@@ -1,26 +1,79 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, { useState } from 'react';
+import styled from "styled-components"
+import {
+  Text,
+  View,
+  FlatList,
+  Alert,
+  TouchableWithoutFeedback,
+  Keyboard
+} from 'react-native';
+
+import Header from './components/Header';
+import TodoItem from './components/TodoItem';
+import AddTodo from './components/addTodo';
 
 const App = () => {
+
+  const [todos, setTodos] = useState([
+    { text: 'Buy wine', key: '1' },
+    { text: 'Learn JS', key: '2' },
+    { text: 'Yoga', key: '3' }
+  ]);
+
+  const pressHandler = (key) => {
+    setTodos(prevTodos => {
+      return prevTodos.filter(todo => todo.key != key);
+    });
+  };
+
+  const submitHandler = (text) => {
+    if (text.length > 5) {
+      setText('');
+      setTodos(prevTodos => {
+        return [
+          { text, key: Math.random().toString() },
+          ...prevTodos
+        ];
+      });
+    } else {
+      Alert.alert('OOPS', 'New todo is to short, try again', [
+        { text: 'Understood', onPress: () => console.log('closed') }
+      ]);
+    }
+  };
+
   return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
+    <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
+      <Container>
+        <Header />
+        <Content>
+          <AddTodo />
+          <List>
+            <FlatList
+              data={todos}
+              renderItem={({ item }) => (
+                <TodoItem item={item} pressHandler={pressHandler} />
+              )}
+            />
+          </List>
+        </Content>
+      </Container>
+    </TouchableWithoutFeedback>
   )
 }
 
+
 const Container = styled.View`
   flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
+  
 `
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+const Content = styled.View`
+paddingTop: 20;
```

Comment:
this is styled components, therefor you should use normal css syntax in here.

`padding-top: 20;`